kubeflow/Makefile

```diff
@@ -281,7 +267,17 @@ acm-kubeflow: hydrate-asm hydrate-kubeflow
 #*****************************************************************************************
 
 .PHONY: clean-build
-clean-build:
+clean-build: validate-values
+	# Dedupe values to ./upstream/manifest
+	kpt cfg set ./upstream/manifests/gcp name $(NAME)
+	kpt cfg set ./upstream/manifests/gcp gcloud.core.project $(PROJECT)
+	kpt cfg set ./upstream/manifests/gcp gcloud.compute.zone $(ZONE)
+	kpt cfg set ./upstream/manifests/gcp location $(LOCATION)
+	kpt cfg set ./upstream/manifests/gcp log-firewalls $(FIREWALL_LOGS)
+
+	kpt cfg set ./upstream/manifests/stacks/gcp name $(NAME)
+	kpt cfg set ./upstream/manifests/stacks/gcp gcloud.core.project $(PROJECT)
```

Comment:
This is creating a problem in the tests.

From logs:

```
# Dedupe values to ./upstream/manifest
2020-10-10T15:21:40.157842566Z  deploy-kf-master-mmbgs-deploy-gcp-gwrzl-pod-5fgkj  step-deploy-gcp                       /bin/sh: 1: kubeflow-ci-deployment: not found
2020-10-10T15:21:40.157838419Z  deploy-kf-master-mmbgs-deploy-gcp-gwrzl-pod-5fgkj  step-deploy-gcp                       /bin/sh: 1: ZONE: not found
2020-10-10T15:21:40.157834159Z  deploy-kf-master-mmbgs-deploy-gcp-gwrzl-pod-5fgkj  step-deploy-gcp                       /bin/sh: 1: us-central1-c: not found
2020-10-10T15:21:40.157829567Z  deploy-kf-master-mmbgs-deploy-gcp-gwrzl-pod-5fgkj  step-deploy-gcp                       /bin/sh: 1: kf-vbp-1010-3dd: not found
2020-10-10T15:21:40.157822154Z  deploy-kf-master-mmbgs-deploy-gcp-gwrzl-pod-5fgkj  step-deploy-gcp                       /bin/sh: 1: [[: not found
```

```
kpt cfg set ./upstream/manifests/gcp gcloud.compute.zone ZONE
set 3 fields
set 84 fields
```

No value of ZONE could also be a problem.